operations/vouchers.md

```diff
@@ -142,22 +147,44 @@ Returns all rate vouchers filtered by [Service](services.md#service), voucher co
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Id` | string | required | Unique identifier of voucher. |
-| `EnterpriseId` | string | required | Unique identifier of the [Enterprise](enterprises.md#enterprise). |
 | `ServiceId` | string | required | Unique identifier of [Service](services.md#service) the voucher belongs to. |
-| `Type` | string [Voucher type](#voucher-type)| required | Type of the voucher. |
-| `Name` | string | required | Internal name of the voucher. |
+| `Name` | string | required, max length 128 characters | Internal name of the voucher. |
 | `CreatedUtc` | string | required | Creation date and time of the voucher in UTC timezone in ISO 8601 format. |
 | `UpdatedUtc` | string | required | Last update date and time of the voucher in UTC timezone in ISO 8601 format. |
-| `ActivityState` | string [Activity state](#activity-state) | required | Whether voucher is active or deleted. |
+| `Type` | [Voucher Type](vouchers.md#voucher-type) | required | Type of the voucher. |
 | `CompanyId` | string | optional | Unique identifier of [Company](companies.md#company) the voucher is related to. |
 | `TravelAgencyId` | string | optional | Unique identifier of [Company](companies.md#company) with [Travel agency contract](companycontracts.md#travel-agency-contract) the voucher is related to. |
-| `ExternalIdentifier` | string | optional, max 255 characters | Identifier of the voucher from external system. |
+| `OccupiableIntervalStartUtc` | string | optional | Start of the time interval, expressed as the timestamp for the start of the first time unit, in UTC timezone ISO 8601 format (or null if the start time should not be updated). |
+| `OccupiableIntervalEndUtc` | string | optional | End of the time interval, expressed as the timestamp for the start of the last time unit, in UTC timezone ISO 8601 format (or null if the end time should not be updated). |
```

Comment:
question: Would it be possible (and easy) to provide voucher-specific explanation for these?

Reply:
@jnv no problem, what should I write there?

Reply:
We have `OccupiableIntervalStartUtc` and `OccupiableIntervalEndUtc` in Voucher Parameters already, so it could be basically the same:

> Start of the time interval in which the voucher can be applied, in UTC timezone ISO 8601 format.

Reply:
[fixed here](https://github.com/MewsSystems/gitbook-connector-api/pull/622/commits/f0f148e90cddf3d62998c05f745f7b84ddf37da5)